CoreUtilities/src/au/gov/asd/tac/constellation/utilities/graphics/IntArray.java

```diff
@@ -133,6 +133,7 @@ public IntArray(final int initialCapacity) {
             throw new IllegalArgumentException("Illegal Capacity: " + initialCapacity);
         }
         this.elementData = new int[initialCapacity];
+        size=0;
```

Comment:
same here

Reply:
See latest changes, I think this is what you meant?

Reply:
Not quite. I meant `size=0` becoming `size = 0`